src/styles/main.scss

```
@@ -10,5 +11,6 @@ body {
   box-sizing: border-box;
```

можливо краще було задати box-sizing  одразу для всіх блоків
* {
  box-sizing: border-box;
}

---

Можливо, незнаю як буде краще, чи зразу до всіх добавляти чи вставляти в html або body

---

```suggestion
  box-sizing: border-box;
```
```suggestion
  box-sizing: border-box;
```
воно так и є, там було згорнуте